dockerfiles/Dockerfile.test-e2e-env

```diff
@@ -0,0 +1,15 @@
+FROM    docker/compose:1.15.0
+
+RUN     apk add -U bash curl
+
+RUN     curl -Ls https://download.docker.com/linux/static/edge/x86_64/docker-17.06.0-ce.tgz | \
```

Comment:
should we put `channel` and `version` in variables so we can easily change it in the future ?